pyspeckit/cubes/SpectralCube.py

```diff
@@ -775,7 +775,7 @@ def fit_a_pixel(iixy):
                 if verbose_level > 1 and ii==0:
                     log.warning("WARNING: using data std() as error.", PyspeckitWarning)
                 sp.error[:] = sp.data[sp.data==sp.data].std()
-            if sp.error is not None and signal_cut > 0:
+            if sp.error is not None and signal_cut > 0 and not all(sp.error == 0):
                 if continuum_map is not None:
                     snr = (sp.data-continuum_map[y,x]) / sp.error
                 else:
```

Comment:
Do you think that lowering the verbose level for that warning on 776 might also be a good alternative solution to the issue?